hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/reload/jvm/tooling/sidecar/DtSidecarWindow.kt

```diff
@@ -89,23 +95,23 @@ fun DtSidecarWindow(
     val x by xAnimatable.asState()
     val y by yAnimatable.asState()
 
-    val height by animateDpAsState(targetValue = maxOf(windowState.size.height, 512.dp))
+    val animationSpec: FiniteAnimationSpec<Float> = if (isSizeAnimationEnabled) SpringSpec() else SnapSpec()
 
     LaunchedEffect(windowState.position) {
-        xAnimatable.animateTo(targetX.value)
+        xAnimatable.animateTo(targetX.value, animationSpec)
```

Comment:
Is the animation on the window position intentionally removed?

Reply:
Yes, since the resizing animation is removed, animation the potion must also be removed, otherwise the tooling window would instantly cover the application window, then move away.

Reply:
But if the target window just gets moved around, then the position animation should still be correct, or am I missing something?

Reply:
You are absolutely correct, `snapTo` is already used in the case I was thinking of, so disabling this animation is not required. Updated.